hazelcast/src/test/java/com/hazelcast/osgi/HazelcastOSGiInstanceTest.java

```diff
@@ -128,10 +123,8 @@ public void equalsReturnsFalseForDifferentOSGiInstancesWithDifferentDelegatedIns
         HazelcastInstance mockHazelcastInstance2 = mock(HazelcastInstance.class);
         HazelcastOSGiService mockService1 = mock(HazelcastOSGiService.class);
         HazelcastOSGiService mockService2 = mock(HazelcastOSGiService.class);
-        HazelcastOSGiInstance hazelcastOSGiInstance1 =
-                HazelcastOSGiTestUtil.createHazelcastOSGiInstance(mockHazelcastInstance1, mockService1);
-        HazelcastOSGiInstance hazelcastOSGiInstance2 =
-                HazelcastOSGiTestUtil.createHazelcastOSGiInstance(mockHazelcastInstance2, mockService1);
+        HazelcastOSGiInstance hazelcastOSGiInstance1 = createHazelcastOSGiInstance(mockHazelcastInstance1, mockService1);
+        HazelcastOSGiInstance hazelcastOSGiInstance2 = createHazelcastOSGiInstance(mockHazelcastInstance2, mockService2);
```

Comment:
The test is name `equalsReturnsFalseForDifferentOSGiInstancesWithDifferentDelegatedInstanceAndDifferentService` but the same service was used twice (`mockService1` instead of `mockService2`).